src/main/resources/static/js/MainWindow.js

```diff
@@ -675,567 +774,571 @@ var initial = 1000;
+		clearInterval(id);
+		document.getElementById("progress").value = 100;
+		done = true;
+	}
 }
 
 /**
  * loads the data needed for the gui
  */
 function main() {
-    loadConfigs();
-    document.getElementById("log").value = "";
+	loadConfigs();
+	document.getElementById("log").value = "";
 
 }
 
 /**
  * loads the data from the frost-server
- * @param fnSucess Called after successfully confirmed URL
+ * 
+ * @param fnSucess
+ *            Called after successfully confirmed URL
  * @returns
  */
 function urlconfirmed(fnSuccess) {
-	//show loader
+	// show loader
 	document.getElementById("loader").style.display = "block";
-	
+
 	var url = document.querySelector("#frostserverurl").value;
 	var mydata = {
-		frostUrl: url
+		frostUrl : url
 	};
-	
+
 	var res;
 	var message;
-	
-    $.ajax({
-        type: "GET",
-        url: "server-check",
-        data: mydata,
-        success: function (response) {
-        	message = response;
-        	if (response === "Server reachable") {  
-        		document.getElementById("serverurlbox").innerText = url;
-                document.getElementById("serverurlbox").href = url;
-                
-                addToLog("FROST-Server: " + url);
-                
-                addToLog("Try to load things ...");
-            	getThings(function() {
-            		//hide loader
-                	document.getElementById("loader").style.display = "none";
-                	if (!(fnSuccess == null)) {
-                		fnSuccess(); 
-                	}
-            	});
-            	
-            	           	
-        	} else {
-        		r = confirm("Can't connect to: " + url  + "\nDetails: " + message);
-            	document.querySelector("#frostserverurl").value = "";
-            	document.getElementById("serverurlbox").innerText = "";
-                document.getElementById("serverurlbox").href = "";
-              //hide loader
-            	document.getElementById("loader").style.display = "none";
-        	}
-        	
-            
-        },
-        error: function (e) {
-            addToLog(e.responseText);
-            document.querySelector("#frostserverurl").value = "";
-            document.getElementById("serverurlbox").innerText = "";
-            document.getElementById("serverurlbox").href = "";
-            //hide loader
-        	document.getElementById("loader").style.display = "none";
-        }
-
-    });
-}
 
+	$.ajax({
+		type : "GET",
+		url : "server-check",
+		data : mydata,
+		success : function(response) {
+			message = response;
+			if (response === "Server reachable") {
+				document.getElementById("serverurlbox").innerText = url;
+				document.getElementById("serverurlbox").href = url;
+
+				addToLog("FROST-Server: " + url);
+
+				addToLog("Try to load things ...");
+				getThings(function() {
+					// hide loader
+					document.getElementById("loader").style.display = "none";
+					if (!(fnSuccess == null)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4781409)